lib/rubocop/cop/style/space_inside_string_interpolation.rb

```diff
@@ -19,25 +19,39 @@ class SpaceInsideStringInterpolation < Cop
         SPACE_MSG = 'Missing space around string interpolation detected.'.freeze
 
         def on_dstr(node)
+          each_style_violation(node) do |final_node, msg|
+            add_offense(final_node, :expression, msg)
+          end
+        end
+
+        private
+
+        def each_style_violation(node)
```

Comment:
I feel such names are a bit misleading, as this method is not really iterating through a collection with `each_something` often implies. I know you've already used them at some places, but I thought it makes sense to mention this.